package.json

```diff
@@ -20,8 +20,5 @@
     "nightwatch": "0.9.12",
     "selenium-download": "^2.0.10",
     "superagent": "3.5.2"
-  },
-  "devDependencies": {
-    "browserstack-local": "1.3.0"
   }
```

Comment:
This is for using the browserstack local plugin, to run them locally. However, the changes make it so that nightwatch uses selenium and chrome directly and you don't have to use browserstack.